### ISSUE TITLE
Quickcast on items does not work with checked Advanced HotKeys

### ISSUE THREAD

**Issue by Agentex21** [ValveSoftware/Dota2-Gameplay#7442]
### Description

Incompatibility when using the new feature with advanced shortcut options. Even putting the shortcuts in Cast or QuickCast the behavior per item does not overlap as expected.

### Example Match ID (and possibly Timestamp)

Dota 2 Options - Try

### Screenshots

![image](https://user-images.githubusercontent.com/14186433/224339226-ec187f11-013c-4352-bcbe-1b5e282438ac.png)

**Comment by shaileshgarg** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/7381

**Comment by shaileshgarg** [comment]
If Advaned Cast/Quickcast is not on, then everything works fine, Item level KeyUp/Down behavior can be configured and they work fine. 
But if this setting is on, then all Items are either KeyUp or Down, changing per item doesn't work.

**Comment by Agentex21** [comment]
Yes, I was aware of that and today I'm already using the new feature. However, the request for improvement would be the operation with the advanced options that would allow the use of autocast (switch enable) in skills of heroes such as BeastMaster, Viper, Huskar etc.

**Comment by shaileshgarg** [comment]
#7538

**Comment by shaileshgarg** [comment]
This was fixed in today's update.
